rel-eng/openshift-master.service

```diff
@@ -2,6 +2,11 @@
 Description=OpenShift Master
 Documentation=https://github.com/openshift/origin
 After=network.target
+After=etcd.service
+Wants=etcd.service
```

Comment:
Pardon my systemd unit file stupidity.  What does this do if the etcd.service doesn't exist? (like in the case of use the embedded one)